lms/resources/_js_config/__init__.py

```diff
@@ -572,6 +572,10 @@ def _config(self):
             config["debug"]["tags"].append(
                 "role:instructor" if self._lti_user.is_instructor else "role:learner"
             )
+            config["user"] = {
+                "display_name": self._lti_user.display_name,
+                "email": self._lti_user.email,
```

Comment:
What is the email address being used for?

Reply:
Nothing at the moment. I added it just because it was available.

I'll remove it for now and we can always add it back if needed.